app-emulation/punes/punes-0.105.ebuild

```diff
@@ -1,30 +1,32 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
+
 inherit autotools xdg-utils
 
 DESCRIPTION="Nintendo Entertainment System (NES) emulator"
 HOMEPAGE="https://github.com/punesemu/puNES"
 SRC_URI="https://github.com/punesemu/puNES/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="cg"
 
 RDEPEND="
+	cg? ( media-gfx/nvidia-cg-toolkit )
 	dev-qt/qtcore:5
 	dev-qt/qtgui:5
 	dev-qt/qtsvg:5
 	dev-qt/qtwidgets:5
-	cg? ( media-gfx/nvidia-cg-toolkit )
 	media-libs/alsa-lib
 	virtual/opengl"
 
 DEPEND="
 	${RDEPEND}
 	virtual/pkgconfig"
```

Comment:
This is also a BDEPEND. Could you rebase against fresh master so we get a green light from the CI bot?

Reply:
@juippis done